README.md

```diff
@@ -5,6 +5,10 @@ We are in the formative steps of building out infrastructure, process and collab
 ## What does the Node.js End User Feedback team do?
 The Node.js End User Feedback team works with Node.js end users to provide a bi-directional feedback channel for the Node.js project through surveys, meetings and outreach.
 
+### Initiatives
+* Support and promote the annual Node. js Foundation User Survey launched every year at Node.js Interactive (usually in October)
```

Comment:
I'm a bit confused on this point, and it varies from my understanding of the outlined goals of the User Feedback Initiative. Could you explain how the User Feedback Initiative will help with the Foundation User Survey?

Additionally, small typo in here: `Node. js Foundation` -> `Node.js Foundation`

Reply:
See #8, #13 #14. Is there an objection?

Reply:
Based on #13, no. I am concerned that this is higher in the list of projects (lending a perceived higher priority, regardless of actual priority) than the initial goal of the Node.js User Feedback Initiative, which was additional efforts rather than the supplementary efforts.